Source/USemLog/Classes/Control/SLControlManager.h

```diff
@@ -30,20 +34,33 @@ class USEMLOG_API ASLControlManager : public AInfo
 	// Check if the manager if initialized
     bool IsInit() const { return bIsInit; }
 
+	// Check if simulation starts
+	bool IsSimStart() const { return bIsSimStart;  }
```

Comment:
https://docs.unrealengine.com/en-US/Programming/Development/CodingStandard/index.html:

`All functions that return a bool should ask a true/false question, such as IsVisible() or ShouldClearBuffer(). `

`Is Sim Start` is not a sentence.

Reply:
It means IsSimulationStarted. I will change that